node/sc/bridgepool/bridge_tx_pool.go

```diff
@@ -329,7 +329,7 @@ func (pool *BridgeTxPool) add(tx *types.Transaction) error {
 	}
 
 	if pool.queue[from] == nil {
-		pool.queue[from] = NewItemSortedMap()
+		pool.queue[from] = NewItemSortedMap(0)
```

Comment:
Is it okay to call `NewItemSortedMap(0)`?

Reply:
Yes. It means there is no limitation.
Before this PR, the itemSortedMap had no limitation. There is additional code to limitate for bridge tx pool.
The bridge tx pool is not related with this PR. So I wanted to retain the code.

Reply:
Using `0` to indicate limitlessness looks a bit confusing to me. 
How about using `math.MaxInt` or declaring a const like `const UnlimitedItemSortedMap = -1`?

Reply:
`UnlimitedItemSortedMap` looks good. :) I will reflect.